internal/servers/controller/handlers/authmethods/oidc.go

```diff
@@ -259,7 +259,7 @@ func (s Service) authenticateOidcCallback(ctx context.Context, req *pbs.Authenti
 		FinalRedirectUrl: finalRedirectUrl,
 	})
 	if err != nil {
-		return errResponse(errors.New(errors.Internal, op, "Error marshaling parameters")), nil
+		return errResponse(errors.New(errors.Internal, op, "Error marshaling parameters after successful callback", errors.WithWrap(err))), nil
```

Comment:
I thought you wanted to log these errors.  Currently this would not be logged because it is sent as a redirect url instead of through the normal way we return error messages.